scripts/build-post-list.js

```diff
@@ -1,5 +1,5 @@
-const { readdirSync, statSync, existsSync, readFileSync, writeFileSync } = require('fs')
-const { resolve, basename } = require('path')
+const { readdir, stat, pathExists, readFile, writeFile } = require('fs-extra')
```

Comment:
Why are we changing this library here?

Reply:
When using the fs module, tests might sometimes fail when performing a write operation because the parent directory may not be created for some reason. fs-extra solves this issue by creating the directory if it doesn't exist, and it also offers additional features that reduce the amount of code needed